webv2/web.go

```diff
@@ -777,98 +777,95 @@ func setParentTable(w http.ResponseWriter, r *http.Request) {
 }
 
 func parentTableHelper(table string, update bool) *TableInterleaveStatus {
-	tableInterleaveStatus := &TableInterleaveStatus{Possible: true}
+	tableInterleaveStatus := &TableInterleaveStatus{
+		Possible: false,
```

Comment:
Two different suggestions showing same parent names: https://drive.google.com/open?id=1Bq1PHW-y9gplCByUFJdP7nS6Hh3ggcjH&authuser=0&resourcekey=0-s3FuRHQq9MyZyjku6UX9cg&usp=drive_link

Reply:
Updated. Please review it.